sms_sponsorship/templates/sms_registration_confirmation_template.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="sms_registration_confirmation" name="SMS registration confirmation">
+        <t t-call="website.layout">
+            <t t-set="title">Thank you for registration!</t>
+            <div class="oe_structure donation_success">
+                <div class="thank_you_container">
+                    <div class="thank_you_image_container">
+                        <img class="img-responsive" style="width: 100%" src="/sms_sponsorship/static/src/img/compassion_thankyou_donation.jpg"/>
+                    </div>
+                    <div class="thank_you_text_container">
+                        <h1>Registration successful !</h1>
+                        <p class="quote">
+                            Thank you for your interest for your registration !
```

Comment:
This text is not right. I would put `Let a new story for <t t-esc="sponsorship.child_id.preferred_name"/> begin! You will soon receive all documentation needed to start the sponsorship in your mailbox.`